kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/src/main/java/__artifactId__/DefaultController.java

```diff
@@ -0,0 +1,19 @@
+package ${groupId}.${artifactId};
```

Comment:
same here, package parameter instead of group and artifact - it can default then to just groupId